spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.describe "Ingredients page", type: :feature do 
+  it "shows list of all ingredients" do 
+    # before (:each) do
+    Salt = Ingredient.create!(name:"Salt", cost:20)
+    Ground_beef=Ingredient.create!(name:"Ground Beef", cost:40)
+    Pepper =Ingredient.create!(name:"Pepper", cost:45)
+    Ginger =Ingredient.create!(name:'Ginger', cost:50)
+  
+     
+       visit "/ingredients"
+    expect(page).to have_content("Salt")
+    expect(page).to have_content("Ground Beef")
+    expect(page).to have_content("Pepper")
+    expect(page).to have_content("Ginger")
+    expect(page).to have_content("20")
+    expect(page).to have_content("40")
+    expect(page).to have_content("45")
+    expect(page).to have_content("50")
+
+    Ingredient.all.each do |ingredient|
+      expect(page).to have_content(ingredient.name)
+      expect(page).to have_content(ingredient.cost)
```

Comment:
Try your best to avoid these ambiguous tests that could potentially give you false positives. When we're saying that we expect 4 to show up on the page, it's ambiguous because it just saying that 4 should show up on the page, it could be the temperature, the date, a phone number, etc.

I would add some string interpolation, to make sure it's clear what 4 is for, and that it's showing up where I want it to, and that I'm targeting the correct thing: `expect(page).to have_content("Cost: 4")`